core/ui/src/main/kotlin/nl/q42/template/ui/theme/AppColorSchemeLight.kt

```diff
@@ -0,0 +1,12 @@
+package nl.q42.template.ui.theme
+
+import androidx.compose.ui.graphics.Color
+
+object AppColorSchemeLight : AppColorScheme {
+    override val buttonText: Color = White
+    override val accent: Color = Purple40
+    override val textPrimary = Black
+    override val surface = White
+    override val surfaceSelected = Color(0xff1b1b1b)
```

Comment:
its a very small nitpic but if we already started making the type explciit for the others, i would expect it to have it for all